packages/commonwealth/client/scripts/views/components/sidebar/governance_section.tsx

```diff
@@ -479,7 +475,7 @@ export class GovernanceSection
       },
       displayData: governanceGroupData,
       isActive: false,
-      toggleDisabled: vnode.attrs.mobile,
+      toggleDisabled: false,
```

Comment:
Does this affect anything?

Reply:
`vnode.attrs.mobile` doesn't exist and hasn't for a while, so `false` is equivalent 🤷 .